test/auto/injectorSpec.js

```diff
@@ -1142,15 +1145,13 @@ describe('strict-di injector', function() {
 
   it('should always use provider as `this` when invoking a factory', function() {
     var called = false;
-    function factoryFn() {
+    function FactoryFn() {
```

Comment:
Could we use `/* @this */` or something instead? It is weird to have a factory function capitalized :grin: